lib/internal/main/eval_string.js

```diff
@@ -23,10 +23,24 @@ const {
 const { addBuiltinLibsToObject } = require('internal/modules/helpers');
 const { getOptionValue } = require('internal/options');
 
+const {
+  codes: {
+    ERR_CODE_INTEGRITY_BLOCKED,
+  },
+} = require('internal/errors');
+
 prepareMainThreadExecution();
 addBuiltinLibsToObject(globalThis, '<eval>');
 markBootstrapComplete();
 
+const { isWindows } = require('internal/util');
+if (isWindows) {
```

Comment:
Is it possible to pull the part which get the codes above so that it is under the isWindows check?  If so it might be good to do that.

Reply:
I see below that this uses a pattern common to what is in other files. So seeing that I think its fine either way for me.